doc/man1/flux-ping.rst

```diff
@@ -40,9 +40,11 @@ OPTIONS
    respectively. Default: send to “*any*”.
 
 **-p, --pad**\ *=N*
-   Include in the payload a string of length *N* bytes. The payload will be
-   echoed back in the response. This option can be used to explore the
-   effect of message size on latency. Default: no padding.
+   Include in the payload a string of length *N* bytes. The payload will
```

Comment:
commit message, intentionally leave out T, E?

Reply:
As you noticed above, `pad_bytes` is also an `int` so `T`, `P`, `E` are not supported. At some point I'll redo all this with the `optparse_get_size_int()` helper suggested by @garlick.